proiect/src/app/Main.java

```diff
@@ -0,0 +1,343 @@
+				switch (i) {
+				case 0: {
+					// Add user
+					User user = new User(scanner);
+					service.addNewUser(user);
+					printDelimiter();
+					System.out.println("Added user:\n" + user);
+					printDelimiter();
+					break;
+				}
+				case 1: {
+					// Add business
+					System.out.println("Business type: [Restaurant], [GroceryShop], [other]");
+					String type = scanner.nextLine();
+					System.out.println("Business name:");
+					String name = scanner.nextLine();
+					Business business = BusinessFactory.makeBusiness(type, name);
+					service.addNewBusiness(business);
+					printDelimiter();
+					System.out.println("Added business:\n" + business);
+					printDelimiter();
+					break;
+				}
+				case 2: {
+					// Add driver
+					Driver driver = new Driver(scanner);
+					service.addNewDriver(driver);
+					printDelimiter();
+					System.out.println("Added driver:\n" + driver);
+					printDelimiter();
+					break;
+				}
+				case 3:
+					// Get user account balance
+					User user = pickUser(scanner, service);
+					printDelimiter();
+					System.out.println("User balance: " + user.getAccountBalance());
+					printDelimiter();
+					break;
+				case 4:
+					// Add founds
+					User userToAddFounds = pickUser(scanner, service);
+					printDelimiter();
+					System.out.println("Current user balance: " + userToAddFounds.getAccountBalance());
+					printDelimiter();
+					System.out.println("Enter the ammount to add");
+					while(!scanner.hasNextDouble()) {
+						scanner.next();
+					}
+					double ammount = scanner.nextDouble();
+					service.addFoundsToUser(userToAddFounds, ammount);
+					printDelimiter();
+					System.out.println("New user balance: " + userToAddFounds.getAccountBalance());
+					printDelimiter();
+					break;
+				case 5:
+					// Add product to business
+					Business business = pickBusiness(scanner, service);
+					Product product = new Product(scanner);
+					service.addProductToBusiness(product, business);
+					printDelimiter();
+					System.out.println("New business products: " + business.getProducts());
+					printDelimiter();
+					break;
+				case 6:
+					// Place order
+					User userThatOrders = pickUser(scanner, service);
+					Business businessToOrderFrom = pickBusiness(scanner, service);
+					OrderFactory newOrderFactory = service.createNewOrderFactory(userThatOrders, businessToOrderFrom);
+					
+					boolean finish = false;
+					while (!finish) {
+						System.out.println("Order maker options: [Add], [Finish]");
+						String response = scanner.nextLine();
+						switch (response) {
+						case "Add":
+							System.out.println("Enter product number:");
+							businessToOrderFrom.printProducts();
+							while(!scanner.hasNextInt()) {
+								scanner.next();
+							}
+							int productIndex = scanner.nextInt();
+							scanner.nextLine();
+							System.out.println("Quantity:");
+							Product productToAdd = businessToOrderFrom.getProducts().get(productIndex); 
```

Comment:
if productIndex does not exist would be nice to throw a custom exception and let the user know, instead of throwing a IndexOutOfBoundsException

Reply:
Resolved in issue #12